### ISSUE TITLE
permettre l'export de BP a une échelle arbitraire

### ISSUE THREAD

**Issue by landryb** [georchestra/cadastrapp#674]
Certains utilisateurs mesurent sur les BP .. donc on devrait pouvoir leur laisser le choix de l'échelle, avec une liste d'échelles configurées coté serveur - #320 a fixé un zoom maximum, donc les échelles sont calculées en fonction de la taille de la parcelle, et dans les BP on se retrouve avec des échelles avec "11m", "32m" qui ne sont pas pratiques sur le terrain.

on devrait pouvoir avoir dans ms2-cadastrapp (comme dans le formulaire standard d'impression mapstore) un sélecteur d'échelle avec une liste de valeurs (en dessous du choix du fond de plan), et par défaut la valeur "automatique" avec le comportement actuel.

@MaelREBOUX @catmorales @jusabatier ca vous intéresse ?

**Comment by MaelREBOUX** [comment]
Au secours !!!!
Kutcher sur des impressions de BP : quelle horreur !

Je propose de rejeter cette demande car avec la version [2.0.0-rc de ms2-cadastrapp](https://github.com/georchestra/mapstore2-cadastrapp/releases/tag/v2.0.0-rc) il y a la possibilité de transformer la sélection de parcelles en annotations.
Donc de pouvoir imprimer ensuite la carte courante avec ces annotations à une échelle fixe.

**Comment by landryb** [comment]
> Au secours !!!!
> Kutcher sur des impressions de BP : quelle horreur !

je suis d'accord sur le fond, mais tu ne peux pas maitriser ce que font tes utilisateurs .... et l'utilisation des annotations n'est pas a la portée de beaucoup d'utilisateurs.
